packages/react-core/src/components/Button/hamburgerIcon.tsx

```diff
@@ -0,0 +1,10 @@
+// Because this is such a specific icon that requires being wrapped in a pf-v[current version]-c-button element,
+// we don't want to export this to consumers nor include it in the react-icons package as a custom icon.
+export const hamburgerIcon = (
+  <svg viewBox="0 0 10 10" className="pf-v6-c-button--hamburger-icon pf-v6-svg" width="1em" height="1em">
```

Comment:
nit - can you import the react-tokens for these classes and use `token.name` to populate the classes instead of hardcoding them?